oxidation/crates/parsec_api_types/src/id.rs

```diff
@@ -267,6 +267,9 @@ impl ToString for UserProfile {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub struct FileDescriptor(pub i32);
```

Comment:
signed value ? what negative and 0 stand for ? (yes I'm asking for more comments ! ^^)

Reply:
`0` is a valid file descriptor :p.
but a negative one ... don't know

Reply:
Do you mean I have to search the information ?